GUI/python source/UMSATSGUI_support.py

```diff
@@ -174,19 +186,21 @@ def destroy_window():
     top_level.destroy()
     top_level = None
 
-def command1_enter(e,display):
-    #print('command1_enter\n')
-    writeCommand(e.widget,display,1,cmd1Text.get())
+
+def command1_enter(e, display):
+    # print('command1_enter\n')
+    write_command(e.widget, display, 1, cmd1Text.get())
 
 
-def command2_enter(e,display):
-    #print('command2_enter\n')
+def command2_enter(e, display):
+
     if serial2 is not None:
-        writeCommand(e.widget,display,2,cmd2Text.get())
+        write_command(e.widget, display, 2, cmd2Text.get())
     else:
-        setupSerial()
+        setup_serial()  # Fill in with parameters for serial port 2.
+
 
-def writeCommand(inputBox,display,can,data):
+def write_command(input_box, display, can, data):
```

Comment:
For some of these function headers, can you also explain what each parameter is for? This one in particular isn't clear.